packages/documentation-spa/docs/onboarding-guide/onboarding-guide.md

```diff
@@ -0,0 +1,283 @@
+
+**Start Microservice**
+
+1. Run npm build:dev to generate a build for dev env and npm build for production build.
+2. Run npm start to run your microservice for dev env.
+
+**Testing**
+
+1. Run npm test to run default tests.
+
+### **API Gateway**
+
+The responsibility of the API gateway is to record “_which service is communicating, with whom, and is it allowed to do so_”. Access Control is implemented on top of the API Gateway which enables the authorization and permission model in the data flow. Also, it is a single source of truth for the entire one platform backend. Websocket support is also provided in the gateway. The supported authorization models are,
+
+
+1. JWT Tokens from auth.redhat.com
+2. API Key
+
+**Tools/Integrations:** User Group microservice
+
+API Gateway - API Gateway handles all the tasks involved in accepting and processing up to hundreds of thousands of concurrent API calls, including traffic management, CORS support, authorization and access control, throttling, monitoring.
+
+**Local Development using docker-compose (recommended)**
+
+1. Copy the .env.example to .env. Change/modify the variable values as required.
+2. Change the URIs in config.json and add/remove the microservices as needed
+3. Run the docker-compose service using docker-compose up api-gateway 
+
+The above command will start all the dependent services from this project. (Make sure any external microservices added in step 2 are running and accessible)
+
+_Note:_ Before starting the gateway, also make sure the microservices in this project are configured properly.
+
+**Running Tests**
+
+npm test
+
+### **Database**
+
+One Platform has been configured to use _MongoDB_ with and for native apps to save time, effort, and enhance response time. For example, unless the App is configured to use a ticketing system with feedback One Platform would store the feedback in a local database.  Developers are encouraged to use non-native databases to ease the portability.
+
+### [CI/CD Pipeline steps](https://jamboard.google.com/d/1Dml55rVlVzUo0VxXLxXISt7jtK0K3AW3GWviNr2WwrY/viewer?f=1) to automate the workflow
+
+1. Developer raises a PR on Master Branch
+
+    1. (*)Run Build Tests and Unit tests
+    2. (*)Code quality checks
+    3. (!)Reviewer reviews the code manually.
+
+1. PR is merged on Master
+
+    4. (*)Build and deploy changes to QA
+    5. (*)Run E2E Test cases
+    6. (!)Manual QA Testing
+
+2. On Successful QA Testing
+
+    7. (!)Send a PR from Master to the specific Release branch (rc/app-name)
+    8. (!)Reviewer does final checks
+
+3. (!)Release PR is merged
+
+    9. (*)Build and deploy to Stage
+
+4. (!)Tag created from a Release branch
+
+    10. (*)Build and deploy to Production
+    11. (*)Run Lighthouse CI
+
+
+#### [Quick deploying a One Platform SPAship Instance to MPaaS](https://docs.google.com/document/d/1P4uM1fjOXUq0Ni5fz1OZ2o1Hm1-f4ZT9WOLNof5vchg/edit#heading=h.jjv3mfsaqnxa)
```

Comment:
```suggestion
#### [Quick deploy a One Platform SPAship Instance to MPaaS](https://docs.google.com/document/d/1P4uM1fjOXUq0Ni5fz1OZ2o1Hm1-f4ZT9WOLNof5vchg/edit#heading=h.jjv3mfsaqnxa)
```
Grammatically it's wrong

Reply:
That's the title for the doc, but I'll change it here